modules/client-configuration/pages/cobbler.adoc

```diff
@@ -421,7 +406,7 @@ Bare metal systems management can be enabled or disabled in the {productname} {w
 
 [NOTE]
 ====
-New systems are added to the organization of the administrator who enabled the bare metal systems management feature.
+New systems are added to the organization of the Administrator who enabled the bare metal systems management feature.
```

Comment:
I think we'd better down-case Administrator everywhere.

Reply:
Might need to go through some other docs as well, in that case.

Reply:
In this case, we probably better do it later.